templates/partial/tabs.html.twig

```diff
@@ -0,0 +1,49 @@
+{% set force = false %}
```

Comment:
Should this be a configuration thing?

Reply:
That was dead code that I've now removed. The tab thing doesn't care about _what_ kind of content it receives; all of that is handled by the calling template. So, for admin views for instance, the tab will necessary be created when there's the edit button. If we want those to be open all the time, we'll just need to add some boilerplate message in the calling template.